TIMEOUT: 100000
VENDOR_ID: 0x2886
PRODUCT_ID: 0x0018
RESPEAKER_RATE: 16000
RESPEAKER_CHANNELS: 6 # must flash 6_channels_firmware.bin first
RESPEAKER_WIDTH: 2
RESPEAKER_INDEX: get_respeaker_device_id()
CHUNK: 1024
chunk_size: 0.5
seconds: 5